PROTOCOL.md

```diff
@@ -100,6 +103,19 @@ By default, the reference implementation stores data files in directories that a
 This directory format is only used to follow existing conventions and is not required by the protocol.
 Actual partition values for a file must be read from the transaction log.
 
+### Change Data Files
+Change data files are stored in a directory at the root of the table named `_change_data`, and represent the changes for the table version they are in. For data with partition values, change data files are stored within the `_change_data` directory in their respective partitions (i.e. `_change_data/part1=value1/...`). Writers can _optionally_ produce these change data files as a consequence of operations that change underlying data, like `UPDATE`, `DELETE`, and `MERGE` operations to a Delta Lake table. Operations that only add new data should not produce separate change files. When available, change data readers should use the change data files instead of computing changes from the underlying data files.
```

Comment:
> Operations that only add new data should not produce separate change files.

If an operation adds no change data files, it must only add new data without deleting or updating any existing data.

Reply:
Not clear on what the suggested change is. It appears to be a restatement of the same thing... also shouldn't the causality be thought of in the other direction?

Instead of 

> If an operation adds no change data files, it must only add new data without deleting or updating any existing data.

Isn't it:

> If an operation adds only new data, without deleting or updating any existing data, it should not produce any change data files

Reply:
@zsxwing let me know what you think about this last one! resolved the other comments in the latest commit otherwise.